dbld/rules

```diff
@@ -1,9 +1,9 @@
 #!/usr/bin/make -f
 
-IMAGES=centos6 centos7 jessie artful xenial trusty devshell
+IMAGES=centos_6 centos_7 jessie artful xenial trusty devshell bionic cosmic buster
 DEFAULT_IMAGE ?= xenial
```

Comment:
again, let's use debian-jessie here and match them up with tokens in packages.txt

this should also be inherited in docker image names, I don't think that would be an issue, but let me know if you think it is.